app/forms/journeys/additional_payments_for_teaching/qualification_details_form.rb

```diff
@@ -93,6 +93,24 @@ def dqt_degree_subjects
 
       private
 
+      def eligible_itt_subject_from_dqt
+        dqt_subjects = answers.early_career_payments_dqt_teacher_record&.itt_subjects&.map(&:to_sym)
+
+        if (dqt_subjects || []).any?
+          overlap = (dqt_subjects & possible_eligible_itt_subjects)
+
+          if overlap.any?
+            overlap.first
+          else
+            :none_of_the_above
+          end
+        end
+      end
```

Comment:
I think this will return subjects that are selectable for the policy but not necessarily ones returned from DQT.

I think what we want to do is see if either policy's dqt record returns a subject that is not `none_of_the_above`

```ruby
dqt_subjects = [
  answers.early_career_payments_dqt_teacher_record.ieligible_itt_subject_for_claim,
  answers.levelling_up_premium_payments_dqt_reacher_record.eligible_itt_subject_for_claim
]

return nil if dqt_subjects.empty?

not_none_of_the_above = dqt_subjects.reject { |subject| subject == :none_of_the_above }

if not_none_of_the_above.any?
  not_none_of_the_above.first
else
  :none_of_the_above
 end
```

I _think_ (caveat I don't understand the policy side of this) that both dqt records should return the same value, however for historical reasons they don't.

Reply:
i've updated to a similar implementation you suggested